cms/admin/utils.py

```diff
@@ -240,13 +240,14 @@ class Media:
     EMPTY_CONTENT_VALUE = _("Empty content")
     LC_SORTED_FIELDS = (models.CharField,)
 
-    _content_obj_cache = {}
     _content_cache_request_hash = None
-    _content_qs_cache = {}
     _content_content_type = None
-    _content_subquery_fields = []
 
     def __init__(self, model, admin_site):
+        self._content_subquery_fields = []
```

Comment:
Good catch. This is very important when creating class variables like these.

Reply:
@marksweb Well, I created the issue myself, being too focused on other stuff (quite nice ORM calls etc.), and only noticed now... Glad we fixed this!